02-documentation/developer-guides/building-a-nextjs-blog-with-squidex.md

```diff
@@ -0,0 +1,588 @@
+
+To be able to add content under the posts schema, you will publish the posts schema. Click _Published_ around the top right corner of the page to publish your schema.
+
+Now your schema is published, you will add sample content to your schema that will be displayed once your blog is built. To add content, click _Content_ on the sidebar and select _posts_ if it is not already selected. Click _New_ to create a new post. If you have some content you want to transfer to Squidex, you can paste it here. If you don't, you can manually enter some placeholders for title, slug and content then click _Save and Publish_ to publish the content you created. Click the back icon and _New_ again to create another sample blog post. Repeat this for as many posts as you like then proceed to the next step.
+
+## Step 3: Setting up your Next.js workspace
+
+Now that you have created your schema on Squidex, you will proceed to set up your Next.js workspace.
+
+Open your terminal and run the following command to install Next.js and its dependencies.
+
+```bash
+npx create-next-app squidex-blog
+```
+
+If you are prompted to confirm that you want to install the create-next-app package, type `y` and press enter to continue with the installation.
+
+Once the Next.js is set up, run the following commands to enter the `squidex-blog` directory and start the development server.
+
+```bash
+cd squidex-blog
+yarn dev
+```
+
+The `yarn dev` command starts a Next.js development server at `http://localhost:3000`. Visit `htttp://localhost:3000` to see your Next.js app. You would see _Welcome to Next.js_ on the page. You have now successfully installed Next.js and are ready to start building your blog.
+
+## Step 4: Create global styles and a Layout component
+
+You would start by creating a `Layout` component that adds a header and footer to all the pages of your blog. Create a `components` folder in `squidex-blog` and create a `layout.js` file in the `squidex-blog/components` folder with the following contents:
+
+```jsx
+import Link from "next/link";
+
+export default function Layout({ children }) {
+  return (
+    <div className="container">
+      <div className="header">
+        <Link href="/">
+          <a className="blog-heading">My Blog</a>
+        </Link>
+      </div>
+      {children}
+      <div className="footer">
+        <div>My Blog</div>
+        <div>(c) 2022 </div>
+      </div>
+    </div>
+  );
+}
+```
+
+This creates a footer and a header with a link to your blog's homepage. The `children` prop allows you to wrap whatever content is added into the `Layout` component in the header and footer.
```

Comment:
"The `children` prop allows you to embed whatever content is added into the `Layout` component between header and footer."
 
 wrap => embed
 in the header and footer => between header and footer